docs/coordinates/representations.rst

```diff
@@ -314,6 +314,138 @@ provides the ``base``::
   <SphericalRepresentation (lon, lat, distance) in (rad, rad, kpc)
       ( 0.0048481,  1.04719246,  1.00000294)>
 
+.. Note:: At present, the differential classes are generally meant to work with
+   first derivatives, but they do not check the units of the inputs to enforce
+   this. Passing in 2nd derivatives, e.g., acceleration values (with
+   acceleration units), will succeed, but any transformations that occur through
+   re-representation of the differential will not necessarily be correct.
```

Comment:
I was thinking along the same lines as @adrn here.  My example is spherical coordinates, where you can e.g. look at http://mathworld.wolfram.com/SphericalCoordinates.html and compare (72) to (75)-(80) - that says to me that the spherical<->cartesian acceleration transform has different unit vectors from the velocity transform.

But I admit I did not look *that* closely at the differential machinery to know if it is accounting for this somehow...?